extensions/amp-story/1.0/amp-story-interactive.css

```diff
@@ -16,31 +16,51 @@
 
 .i-amphtml-story-interactive-container {
   font-family: 'Poppins', sans-serif !important;
+  --animation-time: .3s !important;
```

Comment:
If this can't be overridden please prefix it with `i-amphtml-story`
Should it live with all the other customization variables?

Reply:
We should decide whether all these variables should live in amp-story or if there's a point in moving the ones that are independent of the theming into amp-story-interactive.css. Moving them to the interactive css file would make it so that, if it was made a module, those rules would make the amp-story plugin lighter (by some bytes). However, I'll move these to amp-story.css for consistency for now